src/reducers/SigninReducer.js

```diff
@@ -0,0 +1,37 @@
+import { LOGIN_ATTEMPT, LOGIN_FAILED, LOGIN_SUCCESS } from '../constants';
+
+const initialState = {
```

Comment:
Have the initial state in a different file to avoid repetition since it is being used in different reducers

Reply:
Thank you @lukorito. This has been rectified